app/views/players/show.html.haml

```diff
@@ -15,6 +15,8 @@
       = image_tag "flairs/for_ulven.png"
     - elsif @player.player_name == "Tame My Wild"
       = image_tag "flairs/tame.png"
+    - elsif @player.player_name == "Covid 19 V2"
+      = image_tag "flairs/covid_19_v2.png"
```

Comment:
Thanks for the help! This player has another `elsif` clause around line ~194, could we merge the two please?

Reply:
Absolutely, I'll get right on it!